shuup/front/apps/registration/api.py

```diff
@@ -11,13 +11,20 @@
 from django.contrib.auth import get_user_model
```

Comment:
Don't like this ViewSet here: `shuup/front/apps/registration/api.py`
Now if do more then registration